CMakeLists.txt

```diff
@@ -13,11 +13,18 @@ option(BUILD_APP "Build Chatterino" ON)
 option(BUILD_TESTS "Build the tests for Chatterino" OFF)
 option(USE_SYSTEM_PAJLADA_SETTINGS "Use system pajlada settings library" OFF)
 option(USE_SYSTEM_LIBCOMMUNI "Use system communi library" OFF)
-option(USE_SYSTEM_QT5KEYCHAIN "Use system Qt5Keychain library" OFF)
+option(USE_SYSTEM_QTKEYCHAIN "Use system QtKeychain library" OFF)
 option(USE_PRECOMPILED_HEADERS "Use precompiled headers" ON)
+option(BUILD_WITH_QT6 "Use Qt6 instead of default Qt5" OFF)
 
 option(USE_CONAN "Use conan" OFF)
 
+if (BUILD_WITH_QT6)
+    set(MAJOR_QT_VERSION "6")
+else()
+    set(MAJOR_QT_VERSION "5")
```

Comment:
Should this be strings or numbers? No idea how CMake treats things like this

Reply:
In cmake everything is a string, it doesn't have any other types